pkg/run/run.go

```diff
@@ -128,13 +137,16 @@ func NewCmdRun() *cobra.Command {
 	cmd.Flags().StringVar(&o.mode, "mode", defaultRunMode, "Run mode: Availble: regular, upgrade")
 	cmd.Flags().StringVar(&o.upgradeImage, "upgrade-to-image", defaultUpgradeImage, "Target OpenShift Release Image. Example: oc adm release info 4.11.18 -o jsonpath={.image}")
 	cmd.Flags().StringArrayVar(o.plugins, "plugin", nil, "Override default conformance plugins to use. Can be used multiple times. (default plugins can be reviewed with assets subcommand)")
-	cmd.Flags().StringVar(&o.sonobuoyImage, "sonobuoy-image", fmt.Sprintf("quay.io/ocp-cert/sonobuoy:%s", buildinfo.Version), "Image override for the Sonobuoy worker and aggregator")
+	cmd.Flags().StringVar(&o.sonobuoyImage, "sonobuoy-image", fmt.Sprintf("%s/sonobuoy:%s", defaultToolsRepository, buildinfo.Version), "Image override for the Sonobuoy worker and aggregator")
+	cmd.Flags().StringVar(&o.ToolsImage, "tools-image", fmt.Sprintf("%s", toolsImage), "Image containing plugins to be executed.")
+	cmd.Flags().StringVar(&o.imageRepository, "image-repository", "", "Image repository containing required images test environment.")
```

Comment:
I think we could provide one example, appending to the description. Example we did on `upgrade-to-image`